### DISCUSSION TITLE
Best Practice Analyser

### DISCUSSION THREAD

**Discussion by raptoritsolutions** [KelvinTegelaar/CIPP#340] [Q&A]
Currently I am getting a lot of columns showing "No Data".  Security Defaults Enabled and Microsoft Secure Score bring back information.  It didn't used to do this,  I am now on version 1.5.0

Permissions Check and all Tenants appear properly.

![image](https://user-images.githubusercontent.com/93809581/144769010-2d97c6c0-6d54-4ebb-aa39-8f9f1cd93ef6.png)

**Comment by gavsto** [comment]
Have you forced a refresh of the data? This process occurs automatically at midnight each night.

**Reply by raptoritsolutions** [reply]
Yes have forced a refresh and still the same results.  Results are the same after waiting a few days as well. I just updated the version to 1.5 today. I was on 1.3.

**Reply by raptoritsolutions** [reply]
> Have you forced a refresh of the data? This process occurs automatically at midnight each night.

What could be blocking it?  It worked previously. Still a few days later and I have the same issue.

**Reply by SEI-Mkittok** [reply]
@raptoritsolutions Have you run a permissions check in Configuration settings? I had an issue where azure was being Microsofty and I had to re-apply permissions again for a couple API permissions.

**Reply by KelvinTegelaar** [reply]
I believe this has been solved, exchange token was not working.
